File: src/test/resources/sources/security/c/CommandLine.c
#include <stdlib.h>

int main() {
    const char *command = "ls -la /";

    // Use the system() function to execute the command
    int result = system(command);

    // Check the result of the command execution
    if (result == -1) {
        // An error occurred while trying to execute the command
        perror("Error executing command");
        return EXIT_FAILURE;
    } else {
        // Command executed successfully
        printf("Command executed successfully\n");
        return EXIT_SUCCESS;
    }
}

File: src/test/resources/sources/security/py/CommandLine.py
import os

def execute_command(command):
    try:
        # Execute the command
        result = os.popen(command)

        # Read the output
        output = result.read()

        # Check if the command was successful
        if result.close() is None:
            # Print the output
            print("Output:")
            print(output)
        else:
            # Print error message
            print("Error executing command")
    except Exception as e:
        print("An error occurred:", e)

# Command to execute
command = "ls"

# Call the function to execute the command
execute_command(command)